daemon/inspect_unix.go

```diff
@@ -21,7 +22,7 @@ func setPlatformSpecificContainerFields(container *container.Container, contJSON
 }
 
 // containerInspectPre120 gets containers for pre 1.20 APIs.
-func (daemon *Daemon) containerInspectPre120(name string) (*v1p19.ContainerJSON, error) {
+func (daemon *Daemon) containerInspectPre120(name string, version version.Version) (*v1p19.ContainerJSON, error) {
```

Comment:
Bit unsure about having both a "versioned" function here (`containerInspectPre120`), and a version passed. Perhaps someone has a better/cleaner suggestion?